src/hammer-vlsi/hammer_vlsi/hammer_tool.py

```diff
@@ -960,6 +960,23 @@ def get_pin_assignments(self) -> List[PinAssignment]:
             assigns.append(PinAssignment(pins=pins, side=side, layers=layers, preplaced=preplaced))
         return assigns
 
+    def get_pin_metals(self) -> List[Metal]:
```

Comment:
I'm ok leaving this in, as its an internal function, but as we talked about it this isn't quite useful as is.

Reply:
Is it because we don't capture all of the pins?

Reply:
So this function is not used in this PR so thats why it doesn't need to be here.
It's not useful as written because you want to be able to get the pin metals of an inner module via some `get_hierarchical_setting("MyInnerModule", "vlsi.inputs.pin.assignements")` that will be made in a forthcoming PR.

Reply:
I'll stash this for a future PR.

Reply:
done